srtcore/api.cpp

```diff
@@ -240,7 +240,7 @@ int CUDTUnited::startup()
    CGuard gcinit(m_InitLock);
 
    if (m_iInstanceCount++ > 0)
-      return 0;
+      return 0;   // return 1?
```

Comment:
@ethouris What do you think? Looks like 1 should be returned instead of 0.

Reply:
Not sure what this return value was intended to mean. As far as I understand, it should return 0 normally, and 1 when it has been run before already and therefore the initialization was skipped.